README.md

```diff
@@ -54,56 +54,14 @@
 
 ### Install and run required services
 
-Set up a **postgresql@11** database, **redis** cache and **localstack** server. PostgreSQL and Redis can be started either natively or using Docker. We recommend using Docker.
+Set up a **postgresql@11** database, and **redis** cache. PostgreSQL and Redis using Docker.
 
 #### Starting all services using Docker
 
 ```zsh
-export AWS_ENDPOINT=http://localhost:4566
-export FILE_STORAGE_BUCKET_NAME=localstack-upload
-export AWS_LOG_GROUP_NAME=postmangovsg-beanstalk-localstack
-
 npm run dev:services
 ```
 
-#### Starting postgresql and redis natively
-
-```zsh
-# Install postgres
-brew install postgresql@11
-brew services start postgresql@11
-
-# Create the database
-createdb postmangovsg_dev
-
-# Check if you can connect to the database
-psql -h localhost -p 5432 postmangovsg_dev
-
-# Install redis
-brew install redis
-brew services start redis
-
```

Comment:
these ones cannot remove, must have to run the local server

Reply:
I'm just removing the instructions where Redis and PostgreSQL are run natively.

Still keeping the Dockerfile for running these in the Docker container.

<img width="506" alt="image" src="https://user-images.githubusercontent.com/67887489/231631582-ab267a1a-779e-4546-b7ed-0caa995578d6.png">